subsys/testsuite/Kconfig

```diff
@@ -160,4 +160,17 @@ config TEST_BUSY_SIM
 	  It simulates cpu load by using counter device to generate interrupts
 	  with random intervals and random busy looping in the interrupt.
 
+config LOG_BACKEND_MOCK
```

Comment:
why are you defining this to be part of the testsuite and without any associated code? How can this be used? Maybe put this as part of the test itself if you need that or have it in its own commit and  give some details how this can be used.

Reply:
Have moved this option to Kconfig under the testcase.